app/Http/routes.php

```diff
@@ -27,113 +27,197 @@
-Route::controller('/notifications', 'NotificationController');
-Route::controller('/store', 'StoreController', [
-    'getProduct' => 'store.product',
-    'putRequestNotification' => 'store.request-notification',
-]);
-
-Route::resource('tournaments', 'TournamentsController');
-Route::post('/tournaments/{tournament}/unregister', ['as' => 'tournaments.unregister', 'uses' => 'TournamentsController@unregister']);
-Route::post('/tournaments/{tournament}/register', ['as' => 'tournaments.register', 'uses' => 'TournamentsController@register']);
-
-// Forum controllers
-Route::group(['prefix' => 'forum', 'namespace' => 'Forum'], function () {
-    Route::get('/', ['as' => 'forum.forums.index', 'uses' => 'ForumsController@index']);
-    Route::get('{forums}', ['as' => 'forum.forums.show', 'uses' => 'ForumsController@show']);
-
-    Route::get('t/{topics}', ['as' => 'forum.topics.show', 'uses' => 'TopicsController@show']);
-    Route::post('topics/preview', ['as' => 'forum.topics.preview', 'uses' => 'TopicsController@preview']);
-    Route::post('topics/{topics}/reply', ['as' => 'forum.topics.reply', 'uses' => 'TopicsController@reply']);
-    Route::post('topics/{topics}/lock', ['as' => 'forum.topics.lock', 'uses' => 'TopicsController@lock']);
-    Route::post('topics/{topics}/move', ['as' => 'forum.topics.move', 'uses' => 'TopicsController@move']);
-    Route::post('topics/{topics}/vote-feature', ['as' => 'forum.topics.vote-feature', 'uses' => 'TopicsController@voteFeature']);
-    Route::resource('topics', 'TopicsController', ['only' => ['create', 'store']]);
-
-    Route::resource('forum-covers', 'ForumCoversController', ['only' => ['store', 'update', 'destroy']]);
-    Route::resource('topic-covers', 'TopicCoversController', ['only' => ['store', 'update', 'destroy']]);
-
-    Route::get('p/{posts}', ['as' => 'forum.posts.show', 'uses' => 'PostsController@show']);
-    Route::get('posts/{posts}/raw', ['as' => 'forum.posts.raw', 'uses' => 'PostsController@raw']);
-    Route::resource('posts', 'PostsController', ['only' => ['destroy', 'update', 'edit']]);
+// Users section
+Route::group(['prefix' => 'users', 'as' => 'users.'], function () {
+    Route::post('/check-username-availability', ['as' => 'check-username-availability', 'uses' => 'UsersController@checkUsernameAvailability']);
+    Route::post('/login', ['as' => 'login', 'uses' => 'UsersController@login']);
+    Route::delete('/logout', ['as' => 'logout', 'uses' => 'UsersController@logout']);
+    Route::get('/disabled', ['as' => 'disabled', 'uses' => 'UsersController@disabled']);
+
+    // Authentication section (Temporarily set up as replacement/improvement of config("osu.urls.*"))
+    Route::get('/forgot-password', ['as' => 'forgot-password', function () {
+        return Redirect::to('https://osu.ppy.sh/p/forgot');
+    }]);
+    Route::get('/register', ['as' => 'register', function () {
+        return Redirect::to('https://osu.ppy.sh/p/register');
+    }]);
 });
 
+// Admin section
```

Comment:
should be in main domain as well. Though will give unfortunate effect of having store admin page in main domain :dancer: